### PR TITLE
ALICE Stake Pool Registration

### PR DESCRIPTION
**Pull request by SkiLov3**

### CODE REVIEW

File: registry.json [hunk 1]
```diff
@@ -2924,5 +2924,20 @@
     "telegram": "Hayes7777",
     "operatorName": "Steven",
     "github": ""
+  },
+ {
+    "ticker": "ALICE",
+    "poolId": "1c220012e987c342ec4b4c6cea04501d0cf003459804b0e7018d3c73",
+    "social": {
+      "twitter": "StakeAlice",
```
Comment (Bez625) [inline registry.json:2932]: Please could we see some more activity on social media?

**Review by Bez625 — CHANGES_REQUESTED** [review]

**Comment by SkiLov3** [comment]
Absolutely sorry on call this week and last can do in the next few days.

**Comment by Bez625** [comment]
No worries, maybe add a comment on here when you've had time to be a bit more active and then we can review again.

Thanks!

**Comment by SkiLov3** [comment]
Will do,

I'll circle back in a couple days.

**Comment by SkiLov3** [comment]
Hey Chris,

I have been much more active on Twitter if you would give my registration
another consideration.

**Comment by Bez625** [comment]
Merged

**Pull request merged**